frontend/apps/cloudstorage/webdavapi.lua

```diff
@@ -213,6 +213,10 @@ function WebDavApi:uploadFile(file_url, user, pass, local_path, etag)
         }
     })
     socketutil:reset_timeout()
+    if code == "timeout" or "wantread" or "sink timeout" then
```

Comment:
Maybe just type() ~= "number"?

Reply:
I'd add it in
https://github.com/koreader/koreader/blob/e8ed3b0681ca50c2d1c11bed768e249510b8c6dd/frontend/apps/cloudstorage/webdavapi.lua#L220